.travis.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     - stage: scala3
       name: scala3
       # separate job since only a few modules compile with Scala 3 yet
-      script: jabba install adopt@1.11-0 && jabba use adopt@1.11-0 && sbt -Dakka.build.scalaVersion=3.0 "akka-actor-tests/test:compile" akka-actor-testkit-typed/compile akka-actor-typed/compile akka-discovery/test akka-pki/test:compile akka-protobuf/test:compile akka-protobuf-v3/test:compile akka-slf4j/test:compile akka-stream/compile akka-stream-tests-tck/test akka-coordination/test akka-serialization-jackson/test:compile akka-testkit/test akka-stream-testkit/test akka-remote/compile
+      script: jabba install adopt@1.11-0 && jabba use adopt@1.11-0 && sbt -Dakka.build.scalaVersion=3.0 "akka-actor-tests/test:compile" akka-actor-testkit-typed/compile akka-actor-typed/compile akka-discovery/test akka-pki/test:compile akka-protobuf/test:compile akka-protobuf-v3/test:compile akka-slf4j/test:compile akka-stream/compile akka-stream-tests-tck/test akka-coordination/test akka-serialization-jackson/test:compile akka-testkit/test akka-stream-testkit/test akka-remote/test akka-remote-tests/test
```

Comment:
Now I see that some modules we are running the tests while others we are only `test:compile`. 

Shall we run the tests for them all?

Reply:
if they succeed, then yes :)

Reply:
Do we have tests failures when compiling with Scala 3?

Reply:
No, otherwise travis would fail this job

Reply:
I mean the other modules that are still using `test:compile`. I was wondering if they are so because there are some failures which would surprise me, but we never know. 🤷 

Anyway, I will PR and remove all the `test:compile`.

Reply:
Well, I just read your comment about the object / class and serialization. Yeah, indeed a good example of compilation  causing runtime difference. Programming is not easy. 😞